wot-wg-2023-draft.html

```diff
@@ -206,98 +206,42 @@ <h2>Scope</h2>
-           (<a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#binding-deliverable">Binding Templates</a>):</dt>
-           <dd>Refactor security schemes to use Protocol Binding ontologies for protocol-specific security schemes while also 
-            enabling simpler and inline security declarations
-           </dd>
-          <dt>
-            <strong>Canonicalization and Signing</strong>
-            (<a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#discovery-deliverable">Discovery</a>,
-            <a href="#profiles-deliverable">Profiles</a>):
-          </dt>
-          <dd>
-            Add canonicalization and signing algorithms to support signed Thing Descriptions in the Discovery process.
-          </dd>
-          <dt><strong>Timeseries</strong>
+          <dt><strong>Support WoT Integration</strong>
             (<a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#binding-deliverable">Binding Templates</a>):
-          </dt>
-          <dd>
-            Add support for describing historical values or value changes of affordances to the TD specification in an
-            interoperable way with existing solutions.
-          </dd>
-          <dt>
-            <strong>Connection and Payload Reuse Improvements</strong>
-            (<a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#binding-deliverable">Binding Templates</a>,
-            <a href="#profiles-deliverable">Profiles</a>):
-          </dt>
-          <dd>
-            Add support for describing cases where a connection and subsequent payloads are same or similar.
-          </dd>
-          <dt>
-            <strong>Manageable Actions</strong>
-            (<a href="#thing-description-deliverable">Thing Description</a>,
-            <a href="#profiles-deliverable">Profiles</a>):
-          </dt>
-          <dd>
-            Add support for describing actions that can be managed after their invocation.
+            <a href="#profiles-deliverable">Profiles</a>,
+            <a href="#binding-deliverable">Binding Templates</a>):</dt>
+          <dd>The WG will improve the usability of WoT in other ecosystems and communities. Thus, the WG will define core binding and profiling mechanisms and specify additional protocol and payload bindings, and profiles together with reusable, external security vocabularies.
           </dd>
           <dt>
-            <strong>TD Versioning</strong>
+            <strong>Improve Management of TDs</strong>
             (<a href="#thing-description-deliverable">Thing Description</a>,
             <a href="#discovery-deliverable">Discovery</a>):
           </dt>
           <dd>
-            Explain the semantics on how to version Thing Descriptions.
+            The Thing Descriptions require additional mechanisms around them for their management.
```

Comment:
This is a rather vague description - can we please get it more precise?

Reply:
I wanted to add some intro sentence to each work item. The meaning of this sentence is the following one.